src/node/internal/zlib.d.ts

```diff
@@ -113,3 +115,41 @@ export const CONST_BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_1: number;
 export const CONST_BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_2: number;
 export const CONST_BROTLI_DECODER_ERROR_ALLOC_BLOCK_TYPE_TREES: number;
 export const CONST_BROTLI_DECODER_ERROR_UNREACHABLE: number;
+
+export interface ZlibOptions {
+  flush?: number | undefined;
+  finishFlush?: number | undefined;
+  chunkSize?: number | undefined;
+  windowBits?: number | undefined;
+  level?: number | undefined; // compression only
+  memLevel?: number | undefined; // compression only
+  strategy?: number | undefined; // compression only
+  dictionary?: ArrayBufferView | undefined; // deflate/inflate only, empty dictionary by default
+  info?: boolean | undefined;
+  maxOutputLength?: number | undefined;
+}
+
+type ErrorHandler = (errno: number, code: string, message: string) => void;
+type ProcessHandler = () => void;
+
+export class ZlibStream {
+  public [owner_symbol]: Zlib;
```

Comment:
Before private properties in javascript (`this.#private_property`) were a thing, people used Symbol's to store private properties. Each symbol is unique and therefore is different from each other.

```
const valSymbol = Symbol('hello');

class Hello {
  setAttr(val) {
    this[valSymbol] = val;
  }

  getAttr() { return this[valSymbol]; }
}

Since zlib was implemented before private properties, this symbol is used to store the variable reference to the parent class instance.